bgbb_airflow/pred_airflow_job.py

```diff
@@ -52,6 +52,14 @@ def extract(
     param_prefix,
     model_win=90,
     sample_ids: Union[Tuple, List[int]] = (),
+    first_dims=[
+        "locale",
+        "normalized_channel",
+        "os",
+        "normalized_os_version",
+        "country",
+        "app_version",
```

Comment:
Does it make more sense to have the first or most recent app version in this context?

Reply:
Good point, I think most recent version would be more relevant, and doing a quick query says that only 6% of users have the same app_version over a 90 day period (vs 99+% for these other features). I'll just take this one out since it's not very static and I think it'll be a bit more complicated than simply changing `first` to `last` for this one.